pages/doc/integrations_tas_alerts.md

```diff
@@ -46,251 +84,437 @@ This metric can indicate that TAS is out of container space or that there is a l
-2. Check BBS logs for faults and errors that can indicate issues with BBS.
-3. Try scaling the BBS VM resources up. For example, add more CPUs and memory depending on its `system.cpu`/`system.memory` metrics.
-4. Consider vertically scaling the TAS backing database, if `system.cpu` and `system.memory` metrics for the database instances are high.
+**Troubleshooting**
+1. Check BBS logs for faults and errors that can indicate issues with BBS.
+2. Check BBS VM resources. To find these metrics:
+   1. In your TAS integration, on the **Dashboards** tab, click the **TAS: Job Details** dashboard.
+   2. From the **Job** drop-down menu, select **diego_database**.
+   3. Look at the charts **CPU Usage** and **Memory Usage**.
+3. If resources are low, try scaling them up. For example, add more CPUs and memory depending on the CPU memory metrics.
+4. Check VM resources for the TAS backing database. 
+   
+   To find these metrics, go to the **TAS: Platform MySQL** dashboard to check the health. If resources are low, try scaling them up.
 5. If the above steps do not solve the issue, collect a sample of the Diego Cell logs from the BBS VMs and contact VMware Tanzu Support to troubleshoot further.
 
 ## TAS BBS Time to Run LRP Convergence
 
-Time that the BBS took to run its LRP convergence pass. If the convergence run begins taking too long, apps or Tasks may be crashing without restarting. This symptom can also indicate loss of connectivity to the BBS database.
+Time that the BBS took to run its LRP convergence pass.
+
+If the convergence run begins to take long time, apps or Tasks may be crashing without restarting. This symptom can also indicate loss of connectivity to the BBS database.
+
+**Troubleshooting**
 1. Check BBS logs for errors.
-2. Try vertically scaling the BBS VM resources up. For example, add more CPUs or memory depending on its `system.cpu`/`system.memory` metrics.
-3. Consider vertically scaling the TAS backing database, if `system.cpu` and `system.memory` metrics for the database instances are high.
+2. Try vertically scaling the BBS VM resources up. For example, add more CPUs or memory depending on its CPU and memory
+ metrics.
+   1. In your TAS integration, on the **Dashboards** tab, click the **TAS: Job Details** dashboard.
+   2. From the **Job** drop-down menu, select **diego_database**.
+   3. Look at the charts **CPU Usage** and **Memory Usage**.
+3. Check VM resources for the TAS backing database. 
+    
+   To find these metrics, go to the **TAS: Platform MySQL** dashboard to check the health. If resources are low, try scaling them up.
 4. If that does not solve the issue, pull the BBS logs and contact VMware Tanzu Support for additional troubleshooting.
 
+## TAS BOSH Director Health
+
+The total number of times the BOSH health SLI test suite fails. A failed test suite is one in which any number of tests within the test suite fail. With the default thresholds, the alert will fire after 2 consecutive failures. 
```

Comment:
maybe "fails" -> "has failed" is better?

Reply:
Sure, I'll replace it.